lib/helper/Protractor.js

```diff
@@ -210,7 +210,10 @@ class Protractor extends Helper {
     if (!this.options.keepCookies) {
       await this.browser.manage().deleteAllCookies();
     }
-    await this.browser.executeScript('localStorage.clear();');
+    const url = await this.browser.getCurrentUrl();
+    if (!/data:,/i.test(url)) {
```

Comment:
:+1:

Reply:
Yeah I had this code in my project because I had run into the problem before in chrome, so one less corner case :)